Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -5350,7 +5353,8 @@ def panorama_query_logs_command(args: dict):
     job_id = args.get('query_log_job_id')
     illegal_chars = {'@', '#'}
     ignored_keys = {'entry'}
-
+    show_detail = args.get('show-detail', 'no')
```

Comment:
Why not use the arg-to-boolean function in CSP?

Reply:
Because the API accepts either 'yes' or 'no', and not bool values

Reply:
So you need to handle the case where you receive empty value, cause it will fail in the current code.